openwisp_controller/connection/static/connection/css/batch-command.css

```
@@ -0,0 +1,255 @@
+#main #content .filters-top {
+  margin-bottom: 0.5rem;
+}
+
+/* ==== Stepper CSS ==== */
+.stepper {
+  --step-active-bg: var(--ow-color-primary);
+  --step-active-text: var(--ow-color-white);
+  --step-inactive-bg: var(--ow-color-fg-light);
+  --step-inactive-text: var(--ow-color-fg-dark);
+  --arrow-color: var(--ow-color-fg-dark);
+  display: inline-flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)